oomph-vscode/LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License
```

Comment:
Can you use the MIT license that Oomph is already using and perhaps add your name to it? Alternatively, I can switch to ISC.

Reply:
`ISC` is just a trimmed version of `MIT`, I like it but I'm perfectly fine with switching to MIT. What do you prefer?

Reply:
I guess we'll switch to MIT then, because it's easy as deleting this file from this PR (and optionally adding PurpleMyst to the main LICENSE).